src/api/BundlePush/BundlePusher.ts

```diff
@@ -362,7 +362,7 @@ export class BundlePusher {
 
     // uninstall each module individually
     await this.runSshCommandInRemoteDirectory(sshSession, remoteDirectory, this.defaultRemoteNodehomeCmd + " && " +
-                                              "npm uninstall `ls -1 node_modules | tr '/\n' ' '`");
+                                              "if [ \"$(ls node_modules)\" ]; then npm uninstall `ls -1 node_modules | tr '/\n' ' '`; fi");
```

Comment:
Sorry I missed another case that the node_modules directory itself doesn't exist. This should work:

_if [ -d "node_modules" ] && [ "$(ls node_modules)" ]; then npm uninstall \`ls -1 node_modules | tr '/\n' ' '\`; fi_